ts/component/text-input.ts

```diff
@@ -84,10 +70,20 @@ export class TextInput {
 
     document.addEventListener("keydown", handleEsc);
 
-    this.container.append(label);
-    this.container.appendChild(this.textField);
+    this.container.className = "inline-block flex-auto items-center";
+    var textInputContainer = document.createElement("div");
+    textInputContainer.className =
+      "font-inter font-medium text-12 text-black px-2 py-1 rounded-md ring-2 ring-offset-1 ring-red inline-block flex-auto items-center space-x-2 relative z-40";
```

Comment:
I realize you probably didn't add these, but
* `items-center` is for flex containers, so has no effect here
* `flex-auto` is for flex contents, so has no effect here